### ISSUE TITLE
Text Notifications are sent in the clear - Bug or Feature ?

### ISSUE THREAD

**Issue by cconstab** [atsign-foundation/at_client_sdk#584]
**Describe the bug**
When using the notifcationService you can send the NotificationParams.forText('MESSAGE', toAtsign))

This results in an unencrypted text 'MESSAGE" on the wire under the TLS encryption so the secondary can see the data in the clear.

It is not clear to me why this data should not be encrypted ? It is also not clear to a dev that this is sent not E2E Encrypted currently.

**To Reproduce**
1.  Send forText message
2. Then snoop and capture the TLS packets
3. And then decrypt them using WireShark
4. 

**Expected behavior**
Either make it clear to the dev that this is NOT E2E encrypted and or give and option to be be encrypted.
Or 
Transparently to the dev E2E encrypt the message..

**Screenshots**
Using this code :-  

```
  try {
    await notificationService.notify(NotificationParams.forText(json, toAtsign));
    await notificationService.notify(NotificationParams.forUpdate(key, value: json), onSuccess: (notification) {
      _logger.info('SUCCESS:' + notification.toString());
    }, onError: (notification) {
      _logger.info('ERROR:' + notification.toString());
    });
  } catch (e) {
    _logger.severe(e.toString());
  }
  ```
  
  You can see the encrypted AtKey data and the not encrypted forText data
  
  
![image](https://user-images.githubusercontent.com/6131216/176777420-17a3d70e-784f-4f75-8a5e-9a23669dbe9c.png)

**Additional context**
Dart Code but libs are also available in Flutter

**Comment by cconstab** [comment]
Worthy of some thought.. I think this was a hangover from the early days of the NSS.. At this point i can think of no reason for the data not to be transparently E2E encrypted.. 

Thanks

**Comment by gkc** [comment]
Definitely seems it was intended as a feature back in the dim & distant atPast. Hopefully someone can remember why

**Comment by sitaram-kalluri** [comment]
Duplicate issue: https://github.com/atsign-foundation/at_client_sdk/issues/595

**Comment by sitaram-kalluri** [comment]
Duplicate issue: https://github.com/atsign-foundation/at_client_sdk/issues/595

**Comment by sitaram-kalluri** [comment]
The code changes are completed. Following are the PR's

at_commons: [PR-206](https://github.com/atsign-foundation/at_tools/pull/206)
at_server: [PR-812](https://github.com/atsign-foundation/at_server/pull/812)
at_client_sdk: [PR-611](https://github.com/atsign-foundation/at_client_sdk/pull/611)

**Comment by sitaram-kalluri** [comment]
The changes are merged and published in version at_client 3.0.31
